src/main/webapp/css/style.css

```diff
@@ -22,3 +17,11 @@ h1 {
 .green {
   color: #0F9D58;
 }
+
+.logo {
+  text-decoration: none;
+}
+
+#___gcse_0 {
```

Comment:
Ideally we avoid using this CSS ID which could be changed by the CSE implementation. Perhaps target a div wrapper around the CSE searchbox?